### ISSUE TITLE
Cannot boot server from Powershell (SCNvim) outside of C:\Program Files\SuperCollider-3.12.2

### ISSUE THREAD

**Issue by caseyanderson** [supercollider/supercollider#5989]
## Environment

* SuperCollider version: SuperCollider-3.12.2
* Operating system: Windows 11
* Other details (Qt version, audio driver, etc.): Powershell 7 (SCNvim)

## Steps to reproduce
Below in the command line in any directory that is not `C:\Program Files\SuperCollider-3.12.2`
```supercollider
s.boot;
```

## Expected vs. actual behavior

I am currently running Windows 11 and have SuperCollider-3.12.2 installed. I can run SC fine in the built-in IDE. I have been trying to switch over to SCNvim and seem to only be able to boot the server from the command line (in this case Powershell 7) when I am inside `C:\Program Files\SuperCollider-3.12.2`...See photo below for error when trying to boot the server outside of `C:\Program Files\SuperCollider-3.12.2`

<img width="927" alt="scnvim-cantboot" src="https://user-images.githubusercontent.com/1832169/223002358-8beedb8a-f241-45c0-8c6a-f63ddba83e55.png">

**Comment by caseyanderson** [comment]
Thanks to a helpful response on scsynth.org I was able to identify the fix: `C:\Program Files\SuperCollider-3.12.2` needs to be added to Windows System Path as an Environment Variable. Amusingly enough I had done this but had a typo in my path to SC (I had `C;` instead of `C:` LOL). I will close this issue now.

Here are instructions for the fix should anyone else experience this issue:
1. From Start Menu: `System > Advanced System Settings > Environment Variables > Path > Edit`
2. Click `New`, then add a new line that reads `C:\Program Files\SuperCollider-3.12.2\`, the click OK (etc.)
3. Restart CPU (possibly unnecessary but the changes did not appear to take effect until I restarted)
4. In Powershell or Windows Terminal (if using `nvim` like I am)
* `nvim test.scd`
* `:SCNvimStart`
* (in the file) `s.boot;`

Everything works for me now. Hooray!

IMO this process should be included somewhere in the docs for Windows config of SC. I would be happy to volunteer to write this up and assist in adding it somewhere if that would be helpful. I am also happy to volunteer to help test things on Windows if that is at all helpful.
